src/components/articlecard/ArticleCard.svelte

```diff
@@ -35,7 +36,8 @@
   export let width: string = '100%';
 
   const dispatch = createEventDispatcher();
-  let baseClass = `card-mode card-mode--article`;
+  const readClass = read ? ' articlecard--read' : '';
```

Comment:
For at denne skal kunne ændre sig dynamisk skal det ikke sættes på _baseClass_ men på _cssClass_

Reply:
Tror ikke vi har nogle steder, hvor vi skal kunne ændre den dynamisk uden der er page load fordi det kræver artiklen bliver læst. Eller har du et scenarie, hvor det kunne blive potentielt?

Reply:
Jeg vil mene at når det er en parameter der kommer udefra, så bør den nok kunne opdatere uden nyt pageload.

Det kunne være at man havde en fixed liste i toppen, og alle historierne ned af siden, og så opdaterer liste udseendet hver gang man er "færdig" med en historie.

Reply:
Eller noget så simpelt som en "marker som læst"-knap